designer/client/src/i18n/translations/en.translation.json

```
@@ -41,6 +41,10 @@
       "helpText": "Use sections to split a form. For example, to add a section per applicant. The section title appears above the page title. However, if these titles are the same, the form will only show the page title.",
       "title": "Section (optional)",
       "option": "Select a section"
+    },
+    "repeatTitleField": {
+      "helpText": "Appears in the caption above the page heading",
+      "title": "Question set name"
```

Is it possible to change "Repeat title" to "Question set" in the code?

I know these things often drift over time, but good to match them up now whilst we can

```suggestion
    "questionSetField": {
      "helpText": "Appears in the caption above the page heading",
      "title": "Question set name"
```